src/shogun/features/RandomKitchenSinksDotFeatures.cpp

```diff
@@ -8,6 +8,7 @@
 #include <shogun/features/RandomKitchenSinksDotFeatures.h>
 #include <typeinfo>
 #include <utility>
+#include<iostream>
```

Comment:
that should be removed

Reply:
Sorry, i used that for debugging forgot to remove.

Reply:
debugging with cout is extremely inefficient. Check out gdb, which makes everything much easier

Reply:
SG_DEBUG and if you use it consciously you can even leave it in the code.

Reply:
+1